tests/integration-tests/webhooks/webhooks-test.js

```diff
@@ -66,6 +66,27 @@ describe('Webhooks api', function () {
                 assertDeepWebhookEquality(webhook, createWebhookResponse.body);
             });
         });
+        describe('DELETE /v1/webhooks/:webhook_id', function() {
+            it('Create a webhook -> deleting it -> expecting to get 404 in GET', async function() {
+                const webhook = generateWebhook();
+
+                const insertWebhookResponse = await webhookRequestSender.createWebhook(webhook);
+                expect(insertWebhookResponse.statusCode).to.equal(201);
+                const webhookId = insertWebhookResponse.body.id;
+
+                const deleteWebhookResponse = await webhookRequestSender.deleteWebhook(webhookId);
+                expect(deleteWebhookResponse.statusCode).to.equal(204);
+
+                const getWebhookResponse = await webhookRequestSender.getWebhook(webhookId);
+                expect(getWebhookResponse.statusCode).to.equal(404);
+            });
+            it('Deleting unexisting webhook -> expect 204 status code', async function() {
+                const id = uuid.v4();
+
+                const deleteWebhookResponse = await webhookRequestSender.deleteWebhook(id);
+                expect(deleteWebhookResponse.statusCode).to.equal(204);
+            });
+        });
```

Comment:
Since webhook is closely related to jobs (except `global` ones), it'd be good to have atleast one test where we associate a webhook to a job, delete the webhook, and see the job no longer contains the deleted webhook.

Reply:
I thought it is best to do it when integrating with jobs properly, I don't mind adding it to this PR

what do you think ?

Reply:
we also do not prevent a user to insert a global webhook to a job right now, I have that in mind, I just wanted to handle the integration to jobs in a different PR